02 Requiements/UseCases/UseCase1SendRequest(Fully).md

```diff
@@ -0,0 +1,17 @@
+Extensions | <ul><li>1.A If the Consultant specifies a wall height greater than the defined maximum height</li><br><br> 1. The application displays an error message informing the Consultant of the maximum wall height.<br><br> *The process continues from the 2. step.*</ul><br><br><ul><li> 1.B If the Consultant specifies a wall height smaller than the defined minimum height.</li><br><br> 1. The application displays an error message informing the Consultant of the minimum wall height.<br><br> *The process continues from the 2. step.*</ul><br><br><ul><li> 1.C If the Consultant specifies a wall height with a negative value </li><br><br> 1. The application displays an error message instructing the Consultant to use positive values <br><br> *The process continues from the 2. step.*</ul><br><br><ul><li>1.D If the Consultant specifies a wall height with a symbol</li><br><br> 1. The application displays an error message instructing the Consultant to use numbers.<br><br> *The process continues from the 2. step.* </ul><br><br><ul><li>2.A If the Consultant specifies a wall width smaller than the defined minimum width</li><br><br> 1. The application displays an error message informing the Consultant of the minimum wall width.<br><br> *The process continues from the 3. step.* </ul><br><br><ul><li>2.B If the Consultant specifies a wall width with a negative value</li><br><br> 1. The application displays an error message instructing the Consultant to use positive numbers.<br><br> *The process continues from the 3. step.* </ul><br><br><ul><li>2.C If the Consultant specifies a wall width with a symbol</li><br><br> 1. The application displays an error message instructing the consultant to use numbers.<br><br> *The process continues from the 3. step.* </ul><br><br><ul><li> 4.A If the Consultant specifies an invalid name</li><br><br> 1. The application displays an error message informing the Consultant to use a valid name.<br><br> *The process continues from the 4. step.* </ul><br><br><ul><li> 4.B If the Consultant specifies an invalid email-address</li><br><br> 1. The application displays an error message informing the Consultant to specifies a valid Email.<br><br> *The process continues from the 5. step.* </ul><br><br><ul><li> 4.C If the Consultant specifies an invalid phonenumber</li><br><br> 1. The application displays an error message informing the Consultant to specifies a valid.<br><br> *The process continues from the 6. step.* </ul><br><br><ul><li> 4.D If the Consultant specifies an invalid address</li><br><br> 1. The application displays an error message informing the Consultant to specifies a valid address.<br><br> *The process continues from the 7. step.* </ul><br><br><ul><li> 4.E If the Consultant specifies an invalid ZIP-code</li><br><br>1. The application displays an error message informing the Consultant to specifies a valid ZIP code.<br><br> *The process finishes after the 7th step.* </ul>
```

Comment:
In 4.A you wrote, "The process continues from the 4. step." so in 4.B, 4.C, 4.D, and 4.E you have to write "The process continues from the 4. step." and not "The process continues from the 5. step.", "The process continues from the 6. step.", "The process continues from the 7. step.", and "The process finishes after the 7th step."